src/main/java/org/folio/circulation/domain/validation/ItemMissingValidator.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.support.HttpResult.succeeded;
+
+import java.util.function.Function;
+
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.support.HttpResult;
+import org.folio.circulation.support.ValidationErrorFailure;
+
+public class ItemMissingValidator {
+    private final Function<String, ValidationErrorFailure> itemMissingErrorFunction;
+
+    public ItemMissingValidator(
+      Function<String, ValidationErrorFailure> itemMissingErrorFunction) {
+
+      this.itemMissingErrorFunction = itemMissingErrorFunction;
+    }
+
+    public HttpResult<LoanAndRelatedRecords> refuseWhenItemIsMissing(
+      HttpResult<LoanAndRelatedRecords> result) {
+      return result.failWhen(
+        records -> succeeded(records.getLoan().getItem().isMissing()),
+        loans -> {
+          Item item = loans.getLoan().getItem();
+          String message =
+            String.format("%s (%s) (Barcode:%s) has the item status Missing and cannot be checked out",
```

Comment:
Alternative message:    `String.format("%s (%s) (The status for barcode:%s) is Missing and cannot be checked out"` 
Not sure if it's more confusing or not, but I think it's a bit clearer.

Reply:
Thank you Martin! 

Not that I object to your suggestion but I don't have it in me tonight to deviate from the PO's wording :-)